lib/trino-parquet/src/test/java/io/trino/parquet/reader/flat/TestNullsDecoder.java

```diff
@@ -44,15 +45,16 @@ public class TestNullsDecoder
         MIXED_RANDOM_AND_GROUPED_ARRAY = generateMixedData(r, N, MAX_MIXED_GROUP_SIZE);
     }
 
-    @Test
-    public void testDecoding()
+    @ParameterizedTest
+    @MethodSource("trueFalse")
```

Comment:
Don't use parameterized tests, especially for this. Call the method explicitly with each flag. See `BaseConnectorTest.testMaterializedViewBaseTableGone()` for an example.